### ISSUE TITLE
How to use this for Image Labelling?

### ISSUE THREAD

**Issue by ElisonSherton** [grant-m-s/AstronomicAL#14]
Hi guys!

I was following the `README.md` file but couldn't quite figure out whether this could be used for labelling images and subsequntly building image classification models.

Any help would be appreciated. Thanks! :)

Regards,
Vinayak.

**Comment by grant-m-s** [comment]
Hi Vinayak,

Thanks for the question. So currently AstronomicAL has been implemented for training on tabular data only. I plan to implement image training when I add full PyTorch and Tensorflow models.

However it is possible to do labelling on images for a curated test set, as you can use the "selected source information" to show the image you want (see the docs [here](https://astronomical.readthedocs.io/en/latest/content/tutorials/settings.html#extra-information-columns) ). You will need to create a dataset with a column of IDs, labels, and the corresponding image paths, with a random column (or another data column if you have extras) that would normally be used for training (but in labelling mode you wont be training so its just to prevent any errors where it looks for an empty list).

I hope that helps.

Cheers,
Grant

**Comment by ElisonSherton** [comment]
Thanks Grant, 

I will try it out! :)
